src/index.ts

```diff
@@ -72,6 +72,19 @@ export class WebRTCPlayer extends EventEmitter {
   }
 
   private async onConnectionStateChange(e) {
+
+    if (this.peer.connectionState === 'connected') {
+      const senders = await this.peer.getSenders();
+      if (senders) {
+        senders.forEach(sender => {
+          if (sender.track.readyState === 'ended') {
```

Comment:
sender.track can be null so this actually throws an error for me when I try this code.